src/panels/config/zha/zha-groups-data-table.ts

```diff
@@ -30,6 +30,19 @@ export class ZHAGroupsDataTable extends LitElement {
   @property() public groups: ZHAGroup[] = [];
   @property() public selectable = false;
 
+  private _groups = memoizeOne((groups: ZHAGroup[]) => {
+    let outputGroups: GroupRowData[] = groups;
+
+    outputGroups = outputGroups.map((group) => {
+      return {
+        ...group,
+        id: "" + group.group_id,
```

Comment:
You want it to be a string?
```suggestion
        id: String(group.group_id),
```

Reply:
multi select doesn't work for ids that aren't strings from what I can tell